docs/site/content/en/docs/Concepts/core-concepts/index.md

```diff
@@ -126,6 +126,7 @@ In Horreum a `Report Configuration` is used for compiling a summary of informati
 ## Report
 A `Report` is an instance of a `Report Configuration`. Creation date and time is used for differentiating each `Report`.
 
+<<<<<<< HEAD
```

Comment:
Can you clean this up, looks like a remnant of conflct?

Reply:
well spotted